go/service/chat_local.go

```diff
@@ -63,6 +63,22 @@ func (h *chatLocalHandler) NewConversationLocal(ctx context.Context, trip chat1.
 	return id, err
 }
 
+func (h *chatLocalHandler) CompleteAndCanonicalizeTlfName(ctx context.Context, tlfName string) (res keybase1.CanonicalTlfName, err error) {
+	// Append username in case it's not present. We don't need to check if it
+	// exists already since CryptKeys calls below transforms the TLF name into a
+	// canonical one.
+	tlfName = tlfName + "," + string(h.G().Env.GetUsername())
```

Comment:
are we sure the user is logged in here and that `GetUsername` will always return something useful?